swmmio/tests/data/model_full_features_network_xy.inp

```diff
@@ -312,31 +312,30 @@ C2.1             2747634.497        1117846.256
 C2.1             2747834.998        1117744.149       
 
 [Polygons]
-;;Subcatchment   X-Coord            Y-Coord           
-;;-------------- ------------------ ------------------
-S1               2746954.580        1118179.306       
-S1               2747306.004        1117998.644       
-S1               2747284.350        1118423.694       
-S1               2746979.947        1118463.291       
-S2               2747840.565        1118007.306       
-S2               2748063.299        1118230.039       
-S2               2747405.616        1118921.132       
-S2               2747078.940        1118955.779       
-S2               2747430.364        1118537.535       
-S2               2747492.234        1118089.594       
-S3               2748103.514        1118193.536       
-S3               2747848.608        1117921.306       
-S3               2748088.665        1117782.716       
-S3               2748242.104        1117782.716       
-S3               2748511.859        1117807.464       
-S4               2746913.127        1118252.931       
-S4               2746942.825        1118500.413       
-S4               2747192.781        1118468.241       
-S4               2747168.033        1118599.406       
-S4               2747088.839        1118755.319       
-S4               2746741.127        1118938.456       
-S4               2746527.056        1118812.240       
-S4               2746551.804        1118480.615       
+;;   X             Y            
```

Comment:
Are these changes necessary for anything? If not, let's remove these changes from the PR